__tests__/main.test.ts

```diff
@@ -330,7 +330,9 @@ describe("VIB", () => {
     })
 
     it("Fetches multiple execution graph logs", async () => {
-      jest.setTimeout(20000)
+      jest.setTimeout(300000)
+
+      process.env.INPUT_UPLOAD_ONLY_ON_FAILURE = "false"
```

Comment:
As with the comment below. There is a mismatch between the property name in `action.yml` and docs with what you are using in the code and setting here in tests.